docs/tutorials/aws.md

```diff
@@ -80,18 +81,53 @@ spec:
     spec:
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.2
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.7
         args:
         - --source=service
         - --source=ingress
         - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
         - --provider=aws
         - --policy=upsert-only # would prevent ExternalDNS from deleting any records, omit to enable full synchronization
+        - --aws-zone-type=public #only look at public hosted zones (valid values are public/private)
```

Comment:
nit: maybe note which one is the default value.

Reply:
@mikkeloscar  which is the default?

Reply:
default is both public and private zones.

Reply:
@shavo007 please document the default here